### ISSUE TITLE
RuntimeError: generator raised StopIteration

### ISSUE THREAD

**Issue by edjubuh** [purduesigbots/pros-cli#5]
https://sentry.io/purdue-acm-sigbots/pros-cli-3/issues/673174721/

```
StopIteration: 
  File "lepl\matchers\transform.py", line 202, in _match
    results = yield generator
  File "lepl\core\parser.py", line 146, in trampoline
    value = next(value.generator)

StopIteration: 
  File "lepl\matchers\transform.py", line 204, in _match
    yield function(stream_in, lambda: raise_(StopIteration))
  File "lepl\matchers\derived.py", line 204, in apply
    (results, stream_out) = matcher()
  File "lepl\matchers\transform.py", line 204, in <lambda>
    yield function(stream_in, lambda: raise_(StopIteration))
  File "lepl\matchers\transform.py", line 167, in raise_
    raise e

RuntimeError: generator raised StopIteration
(17 additional frame(s) were not displayed)
...
  File "lepl\regexp\rewriters.py", line 331, in clone_regexp
    pattern = Sequence(alphabet_, *alphabet_.parse(pattern))
  File "lepl\regexp\str.py", line 366, in parse
    return self._parser(regexp)
  File "lepl\core\config.py", line 844, in parse_string
    return self.get_parse_string()(string, **kargs)
  File "lepl\core\parser.py", line 257, in single
    return next(raw(arg, **kargs))[0]
  File "lepl\core\parser.py", line 161, in trampoline
    value = stack[-1].generator.throw(value)

RuntimeError: generator raised StopIteration
```

**Comment by edjubuh** [comment]
Closed by 72085403c738d918ae7159643a6d6841f033b658

**Comment by edjubuh** [comment]
Workaround available:

1. Download the template. If you get the failure when downloading the kernel, download https://pros.cs.purdue.edu/v5/releases/kernel@3.1.0.zip . If you get the failure when downloading okapilib, download https://github.com/OkapiLib/OkapiLib/releases/download/v3.1.0/okapilib.3.1.0.zip
2. Open command prompt and run `prosv5 conduct fetch path/to/kernel@3.1.0.zip` (or `path/to/okapilib.3.1.0.zip`), replacing `path/to/kernel@3.1.0.zip` with the appropriate path.
3. Create PROS project as before.
